### DISCUSSION TITLE
Issue with handling of swap in Wallet Transaction with swapExactTokensForETH

### DISCUSSION THREAD

**Discussion by DF-pop** [pwillworth/dfkreport#28] [General]
Your tool has been very helpful exporting data from Harmony and DFK chain to Koinly and greatly appreciate it.
I noticed a issue, so I just want to report it.
When there is wallet transfer event as swapExactTokensForETH in Harmony Explore, it is not recording swap correctly.
As a example 0x45f1d833ad24a5e8d0e76035b67e89440bac24357b3733c92e9ad38eb8f50319

In only export 1st coin, but 2nd coin doesn't show up in the CSV file.

**Comment by DF-pop** [comment]
I also notice same issue with removeLiquidityETH operation as well

**Reply by pwillworth** [reply]
Hey, thanks for the report.  The way Lila's Ledger works, it needs to be aware of what addresses are dexes swaps happen on ahead of time and in the case of this transaction, I didn't have whatever dex it happened on in the list.  Which caused it to not process right resulting in what you saw.  I can get it added so it shows up right, but if you could let me know what exchange that swap happened on I would appreciate it so I can label it right.

**Reply by DF-pop** [reply]
Hello,
Sure I see the swapExactTokensForETH generated from OpenSwap
https://app.openswap.one/ and I also see swap address for Sushi router address 
0x1b02dA8Cb0d097eB8D57A175b88c7D8b47997506

I don't see any issue though if transaction is showing as swapExactTokensForTokens though for OpenSwap transactions.
